common/v2/features/Home/routes.ts

```diff
@@ -6,5 +6,11 @@ export default [
     path: '/',
     exact: true,
     component: Home
+  },
+  {
+    name: 'Home',
+    path: '/home',
+    exact: true,
+    component: Home
```

Comment:
Why do we need to add this when `/` already routes to `Home`?

Reply:
This is a requirement from Jira; we can also remove that.

Reply:
We haven't decided how to handle routing yet, though it's easy enough to change later, so this is fine.